ipvs.go

```diff
@@ -199,23 +199,37 @@ func parseIPVSBackendStatus(file io.Reader) ([]IPVSBackendStatus, error) {
 	return status, nil
 }
 
-func parseIPPort(s string) (net.IP, uint16, error) {
-	tmp := strings.SplitN(s, ":", 2)
-
-	if len(tmp) != 2 {
-		return nil, 0, fmt.Errorf("invalid IP:Port: %s", s)
-	}
+func parseIPPort(ipPort string) (net.IP, uint16, error) {
```

Comment:
Given the function name is clear, it's more idiomatic to use `s` as variable name.